README.md

```diff
@@ -12,10 +12,10 @@ methods to define the parameters sent to APIv7.
 
 ## Requirements
 
-    - [angular](https://angularjs.org/)
-    - [angular-resource](https://angularjs.org/)
-    - [lodash](https://lodash.com/)
-    - externally provided auth and token injection for OVH APIv6.
+  - [angular](https://angularjs.org/)
+  - [angular-resource](https://angularjs.org/)
```

Comment:
Please use this following link: https://docs.angularjs.org/api/ngResource/service/$resource